src/napari/_qt/layer_controls/widgets/qt_face_color.py

```diff
@@ -0,0 +1,81 @@
+from typing import Optional
+
+import numpy as np
+from qtpy.QtCore import Slot
+from qtpy.QtWidgets import QWidget
+
+from napari._qt.layer_controls.widgets.qt_widget_controls_base import (
+    QtWidgetControlsBase,
+    QtWrappedLabel,
+)
+from napari._qt.utils import qt_signals_blocked
+from napari._qt.widgets.qt_color_swatch import QColorSwatchEdit
+from napari.layers.base.base import Layer
+from napari.utils.translations import trans
+
+
+class QtFaceColorControl(QtWidgetControlsBase):
+    """
+    Class that wraps the connection of events/signals between the current face
+    color layer attribute and Qt widgets.
+
+    Parameters
+    ----------
+    parent: qtpy.QtWidgets.QWidget
+        An instance of QWidget that will be used as widgets parent
+    layer : napari.layers.Layer
```

Comment:
`Image` or `Surface`?

Reply:
I think this widget is used by `Shapes` and `Points` layers, so:
```suggestion
    layer : napari.layers.Shapes | napari.layers.Points
```